packages/mobile/src/components/offline-downloads/DownloadToggle.tsx

```diff
@@ -186,7 +186,7 @@ export const DownloadToggle = ({
         <Switch
           value={isCollectionMarkedForDownload}
           onValueChange={handleToggleDownload}
-          disabled={disabled}
+          disabled={(!collectionId && !isFavoritesDownload) || disabled}
```

Comment:
Did removing this break something? I took it out cause we had the check above where we return null in this case on 158.

Reply:
Yeah lmk what you think, wasn't sure if a bug cause the favorites switch was unresponsive otherwise

Reply:
Interesting. I did check in a change to main to fix this, so maybe try again without? It used to just be `!collectionId || disabled` which was unresponsive.